### DISCUSSION TITLE
年轻人的第一个 Gazelle 音乐站

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#14] [Announcements]
# 年轻人的第一个 Gazelle 音乐站

专而精，精而优，你值得拥有。

https://zexwoo.blog/posts/knowledge/gazelle-music-tracker/

**Comment by aprils148** [comment]
如何知道下次開放申請時間?

**Reply by ZexWoo** [reply]
每年六月一日是站庆，固定会开。

**Comment by candydre** [comment]
想问一下海豚站内的同人音乐活跃度如何？

**Reply by ZexWoo** [reply]
有一些爱好者，也有一些独占资源。

**Comment by dumbbird** [comment]
你好，很高兴看到有这样的网站和爱好者群体。很有兴趣加入并看看是否能提供一些贡献。
不过我比较感兴趣的是关于wiki百科的内容，想问问看这些百科里的内容是只集中在技术层面（如何抓轨、如何辨别版本等等），还是可以包括细致到具体某一个艺术家、某一张专辑、某一首单曲的音乐本质层面上的信息。

提出这个问题的原因是，目前我也在fandom上经营一个类似discogs的wiki站，主要是想分享一些细致到专辑、单曲层面上的音乐信息，包括唱片catelog，制作团队credits，不同的版本之间的差异——音质上的以及包装上的，都可以。所以，不知道贵站是否有兴趣，可以在某些领域上达成一些合作。谢谢！

附，本人在fandom上的wiki sample页：https://discogs.fandom.com/zh/wiki/%E9%83%BD%E6%98%AF%E5%A4%9C%E5%BD%92%E4%BA%BA_(%E8%AE%B8%E7%BE%8E%E9%9D%99%E4%B8%93%E8%BE%91)

**Reply by ZexWoo** [reply]
请你加下 Q 群 370321105，申请文本里面除了回答问题，再备注一下 fandom 的字样，我看到了以后会联系你。

**Reply by dumbbird** [reply]
很久不用qq了，请问能否有其他的方式交流，微信可以么？

**Reply by ZexWoo** [reply]
发邮件给我吧。zexwoo@qq.com

**Reply by dumbbird** [reply]
已发邮件给你～:)
